docs/getting-started.md

```diff
@@ -27,7 +27,7 @@ If you are adding `modguard` to an existing project, you have two main options:
 
 ```bash
 # From the root of your Python project
-modguard .
+modguard check .
 ```
 
-After guarding your project, running `modguard` from the root will check all imports to verify that packages remain correctly decoupled.
+After implementing in your project, running `modguard` from the root will check all imports to verify that packages remain correctly decoupled.
```

Comment:
Given the title of the section, maybe we keep the original? Or switch to 'initializing'?